packages/tupaia-web/src/features/PDFExportDashboardItem/PDFExportDashboardItem.tsx

```diff
@@ -161,6 +168,7 @@ export const PDFExportDashboardItem = ({
 
   const title = entityHeader ? `${name}, ${entityHeader}` : name;
   const period = getDatesAsString(periodGranularity, startDate, endDate);
+  const exportDescription = settings?.exportDescription;
```

Comment:
```suggestion 
```
Personally I'm not a fan of setting variables like this. If we know `settings` exists we can destructure `exportDescription` from it, but I think in this case we can directly use `settings?.exportDescription` below